### ISSUE TITLE
Don't work GetAvailableRooms

### ISSUE THREAD

**Issue by SergeyMaas** [gamestdio/colyseus-unity3d#42]
Hi!!!
Why in Client.cs commented code part with function GetAvailableRooms?
When that will be fixed and i can work with that?

```
//		public void GetAvailableRooms (string roomName, Action<RoomAvailable[]> callback)
//		{
//			int requestId = ++this.requestId;
//			this.connection.Send (new object[]{Protocol.ROOM_LIST, requestId, roomName});
//
//			this.roomsAvailableRequests.Add (requestId, callback);
//
//			// // USAGE
//			// this.client.GetAvailableRooms ("chat", (RoomAvailable[] obj) => {
//			// 	for (int i = 0; i < obj.Length; i++) {
//			// 		Debug.Log (obj [i].roomId);
//			// 		Debug.Log (obj [i].clients);
//			// 		Debug.Log (obj [i].maxClients);
//			// 		Debug.Log (obj [i].metadata);
//			// 	}
//			});
//		}
```

**Comment by endel** [comment]
Hi @SergeyMaas,

As far as I remember, this callback wasn't being triggered properly:

https://github.com/gamestdio/colyseus-unity3d/blob/778159e572b8babcd848ba014d047cf966e6e58c/Assets/Plugins/Colyseus/Client.cs#L204-L208
That's why I left the whole `GetAvailableRooms` functionality commented out. I'd appreciate pull-requests to fix this issue!

Cheers

**Comment by bhavik66** [comment]
@endel @SergeyMaas Pull request created to fix this issue. Thanks

**Comment by endel** [comment]
Thanks a lot, @bhavik66! Just released [0.9.5](https://github.com/gamestdio/colyseus-unity3d/releases/tag/0.9.5) with your pull-request!
